src/test/java/com/sforce/cd/apexUnit/fileReader/ApexManifestFileReaderTest.java

```diff
@@ -59,7 +59,7 @@
 	ApexClassFetcherUtils apexClassFetcher = new ApexClassFetcherUtils();
 	int limit = 10;
 	String[] apexClasses = new String[limit];
-	ApexManifestFileReader apexManifestFileReader = new ApexManifestFileReader();
+	ApexManifestFileReader apexManifestFileReader = new ApexManifestFileReader(true);
```

Comment:
would be nice to add a comment indicating the purpose of boolean parameter and why the boolean parameter is being passed as true